src/hypertable.c

```diff
@@ -1905,6 +1905,38 @@ ts_hypertable_distributed_create(PG_FUNCTION_ARGS)
 	return ts_hypertable_create_internal(fcinfo, true);
 }
 
+/* Go through columns of parent table and check for column data types. */
+static void
+ts_validate_basetable_columns(Relation *rel)
+{
+	int attno;
+	TupleDesc tupdesc;
+
+	tupdesc = RelationGetDescr(*rel);
+	for (attno = 1; attno <= tupdesc->natts; attno++)
+	{
+		Form_pg_attribute attr = TupleDescAttr(tupdesc, attno - 1);
+		Oid typid = attr->atttypid;
+		switch (typid)
+		{
+			case CHAROID:
+			case VARCHAROID:
+				ereport(NOTICE,
```

Comment:
Shouldn't those messages bet sent as WARNING instead of NOTICE?

Reply:
Fixed.